Pcap++/src/DpdkDeviceList.cpp

```diff
@@ -56,6 +56,7 @@ namespace pcpp
 bool DpdkDeviceList::m_IsDpdkInitialized = false;
 CoreMask DpdkDeviceList::m_CoreMask = 0;
 uint32_t DpdkDeviceList::m_MBufPoolSizePerDevice = 0;
+bool DpdkDeviceList::m_DpdkInitVerify = true;
```

Comment:
Why do we need it as a private member of `DpdkDeviceList`? Instead, we can add it as an additional argument to `DpdkDeviceList::initDpdk()` because it's only used there... 🤔